roles/openshift_metering/files/monitoring/rbac/role.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
Can you explain the necessity of this role and rolebinding?

Reply:
It's the same as what exists in cluster-monitoring here: https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/prometheus-k8s/role-binding-specific-namespaces.yaml which gives prometheus-operator/prom access to query resources in the openshift-metering namespace.